### PR TITLE
[MIG] sale-open-qty: Migration to 12.0

### PR DESCRIPTION
**Pull request by andreampiovesana**
migration to v12 with dependent module sale_order_line_input

### CODE REVIEW

**Comment by rousseldenis** [comment]
> migration to v12 with dependent module sale_order_line_input

Hi @andreampiovesana. Thanks for your contribution. Please keep commit history following https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

**Comment by rousseldenis** [comment]
@MiquelRForgeFlow Apparently your module has success ! :smiley:

**Comment by MiquelRForgeFlow** [comment]
Oh, nice.  😁

**Comment by andreampiovesana** [comment]
> > migration to v12 with dependent module sale_order_line_input
> 
> Hi @andreampiovesana. Thanks for your contribution. Please keep commit history following https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

can you help me with istruction to do?

File: sale_open_qty/README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Miquel Raïch <miquel.raich@eficent.com>
```
Comment (MiquelRForgeFlow) [inline sale_open_qty/README.rst]:
```suggestion
* Miquel Raïch <miquel.raich@forgeflow.com>
```
Also please, change in all headers Eficent to ForgeFlow S.L.

**Comment by MiquelRForgeFlow** [comment]
> can you help me with istruction to do?

In a new branch, or just in this one but first reset it to 12.0 (you may want to copy the module folder first, then after recreating commit history, then you paste it again doing the migration commit again), do the following in the shell command:

`git format-patch --keep-subject --stdout origin/12.0..origin/9.0 --  sale_open_qty | git am -3 --keep`

**Comment by andreampiovesana** [comment]
can you do it for me?

Andrea Piovesana Linux Mint 19.3www.pnlug.it



Il giorno mar 28 lug 2020 alle ore 18:15 Miquel Raïch <
notifications@github.com> ha scritto:

**Comment by andreampiovesana** [comment]
to keep commit history

**Pull request closed without merging**